<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>camers</class>
 <widget class="QMainWindow" name="camers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>camers</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="list_cams">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="species_output">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Здесь будет вид</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose_cur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать выделенный элемент</string>
    </property>
   </widget>
   <widget class="QLabel" name="age_output">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Здесь будет возраст</string>
    </property>
   </widget>
   <widget class="QLabel" name="vent_text">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Вентиляция:</string>
    </property>
   </widget>
   <widget class="QLabel" name="vent_output">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>vent</string>
    </property>
   </widget>
   <widget class="QLabel" name="food_text">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Еда:</string>
    </property>
   </widget>
   <widget class="QLabel" name="food_output">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>food</string>
    </property>
   </widget>
   <widget class="QLabel" name="clear_text">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Чистота:</string>
    </property>
   </widget>
   <widget class="QLabel" name="clear_output">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>260</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>default</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vent_fixat">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Исправить вентиляцию</string>
    </property>
   </widget>
   <widget class="QPushButton" name="food_fixat">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Доложить еду</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_fixat">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>260</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sohran_v_file">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>450</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить в файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_el_to_list">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>450</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить новый элемент
из C:/data/new_el</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del_el_from_l">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>450</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить выбранный элемент</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>923</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
